### DISCUSSION TITLE
Toggle Snap in QField

### DISCUSSION THREAD

**Discussion by Oisin-JK** [opengisch/QField#3091] [Ideas]
It would be nice to be able to toggle snap on and off from Qfield.

**Comment by lucienicolier** [comment]
Hi,

It is currently possible. You need to set up it in your QGIS project.

Have a look in the doc [topological editing](https://docs.qfield.org/how-to/digitize/#topological-editing)

**Reply by Oisin-JK** [reply]
Hi
Maybe you have mis-understood me? I know how set up snapping in QGIS for QField, and that works. But in QField, it would be nice to be able to turn snapping off (and on again). Does that make sense?

**Reply by geomondego** [reply]
@Oisin-JK To do this, simply click on the respective button on the left under the menu button
![Screenshot from 2022-07-26 18-30-59](https://user-images.githubusercontent.com/4389239/181072785-c2994278-1f40-4b63-9ac5-c09778597d63.png)

**Reply by Oisin-JK** [reply]
Hi
But this symbol toggles topological snapping, not snapping. And it isn't visible when on a point layer.

**Reply by lp-dj** [reply]
Yes, I have to agree @Oisin-JK .

Topological editing

(„shared nodes of neighbouring geometries are modified together when moving / deleting vertices and additional vertices are added to segments when a new node is added from neighbouring geometries“ quote from https://docs.qfield.org/how-to/digitize/?h=topologic#topological-editing ) 

 is not necessarily the same as snapping

( let QGIS \ Qfield search for the closest vertex or segment you are trying to connect to within the given search radius while digitizing a feature).

Would also find an additional button to turn snapping on & off in QField very handy.
